doc/reference/tt_cli/cat.rst

```diff
@@ -0,0 +1,56 @@
+Printing the contents of .snap and .xlog files
+==============================================
+
+..  code-block:: bash
+
+    tt cat FILE .. [flags]
+
+``tt cat`` prints the contents of :ref:`snapshot <internals-snapshot>` (``.snap``) and
+:ref:`WAL <internals-wal>` (``.xlog``) files to stdout. A single call of ``tt cat`` can
+print the contents of multiple files.
+
+
+Flags
+-----
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 20 80
+        :header-rows: 0
+
+        *   -   ``--format``
+            -   Output format: ``yaml`` (default), ``json``, or ``lua``
+        *   -   ``--from``
+            -   Show operations starting from the given LSN
+        *   -   ``--to``
```

Comment:
I think it's worth mentioning that `from` and `to` only make sense for xlogs, not snaps (I believe it's so but I'm not 100% sure).
Also they only make sense when **one** replica id is passed as far as I understand

Reply:
As discussed in DM, `from` and `to` do make sense for snaps and this may be useful. Marking this as resolved.

Reply:
What about the second comment?
> Also they only make sense when one replica id is passed as far as I understand. Otherwise it's unclear whether cat should start with replica_id1:from_lsn or replica_id2:from_lsn

Reply:
Added this to *Details* subsections on play and cat pages.

Reply:
Thanks!